src/responder/kcm/kcmsrv_ccache_mem.c

```diff
@@ -456,11 +467,20 @@ static struct tevent_req *ccdb_mem_getbyname_send(TALLOC_CTX *mem_ctx,
     ccwrap = memdb_get_by_name(memdb, client, name);
     if (ccwrap != NULL) {
         state->cc = kcm_ccache_dup(state, ccwrap->cc);
+        if (state->cc == NULL) {
+            ret = ENOMEM;
+            goto fail;
```

Comment:
Same comment about using goto end/finish applies here as well.

Reply:
Also here the suggestion was taken, you are right it is more idiomatic to use a single label in the send function.